server.js

```diff
@@ -23,10 +13,79 @@ app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
+// request is incoming an res is outgoing
+// used listEndpoints to list all routes
+
+const ERROR_MESSAGE = 'No such book found'
```

Comment:
Good idea with predefined error message! Also, I'm happy to see proper naming convention (uppercase letters divided by `_`)